configs/defaults/large_cohort.toml

```diff
@@ -6,6 +6,8 @@ status_reporter = 'metamist'
 # define reference fasta. If not specified here, the reference file is pulled from [references.broad][ref_fasta]
 ref_fasta='gs://cpg-common-main/references/hg38/v0/dragen_reference/Homo_sapiens_assembly38_masked.fasta'
 highmem_workers = true
+highmem_drivers = true
```

Comment:
I think this default behaviour is up to the PA team. If you are inherently going to need highmem for larger cohorts, then it makes sense to keep it as the default. I guess the consideration is whether this is overkill/drives up cost for smaller test runs.

Reply:
I'll remove this as the default behaviour given how infrequently we run complete cohorts. I've put it to the rest of PA to discuss what the default behaviour should be. Thanks @vivbak !